example/plotOptions/bundle.js

```diff
@@ -0,0 +1,145 @@
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define*/
+
+define([
+    'legacyRegistry',
+    '../../platform/commonUI/browse/src/InspectorRegion'
+], function (
+    legacyRegistry,
+    InspectorRegion
+) {
+    "use strict";
+
+    /**
+     * Add a 'plot options' region part to the inspector region for the
+     * Telemetry Plot type only. {@link InspectorRegion} is a default region
+     * implementation that is added automatically to all types. In order to
+     * customize what appears in the inspector region, you can start from a
+     * blank slate by using Region, or customize the default inspector
+     * region by using {@link InspectorRegion}.
+     */
+    var plotInspector = new InspectorRegion(),
+     /**
+      * Two region parts are defined here. One that appears only in browse
+      * mode, and one that appears only in edit mode. For not they both point
+      * to the same representation, but a different key could be used here to
+      * include a customized representation for edit mode.
+      */
+        plotOptionsBrowsePart = {
+            name: "plot-options",
+            title: "Plot Options",
+            modes: ['browse'],
+            content: {
+                key: "plot-options-browse"
+            }
+        },
+        plotOptionsEditPart = {
+            name: "plot-options",
+            title: "Plot Options",
+            modes: ['edit'],
+            content: {
+                key: "plot-options-browse"
+            }
+        };
+
+    /**
+     * Both parts are added, and policies of type 'region' will determine
+     * which is shown based on domain object state. A default policy is
+     * provided which will check the 'modes' attribute of the region part
+     * definition.
+     */
+    plotInspector.addPart(plotOptionsBrowsePart);
+    plotInspector.addPart(plotOptionsEditPart);
+
+    legacyRegistry.register("example/plotType", {
+        "name": "Plot Type",
+        "description": "Example illustrating registration of a new object type",
+        "extensions": {
+            "types": [
+                {
+                    "key": "plot",
+                    "name": "Telemetry Plot",
+                    "glyph": "t",
+                    "description": "A plot for displaying telemetry",
+                    "delegates": [
+                        "telemetry"
+                    ],
+                    "features": "creation",
+                    "contains": [
+                        {
+                            "has": "telemetry"
+                        }
+                    ],
+                    "model": {
+                        "composition": []
+                    },
+                    "regions": {
+                        "inspector": plotInspector
+                    },
+                    "telemetry": {
```

Comment:
Originally had a blank plot, but added the sine wave config so that the plot shows something. Blank plot would be ok if telemetry could be dropped onto it, but currently that would result in a new panel which is not configured for new plot options.

Can remove sine wave stuff if a blank plot is preferable here?